cli/serve/serve.go

```diff
@@ -153,6 +157,10 @@ var endpoints = map[string]func() (http.Handler, error){
 	},
 
 	"/": func() (http.Handler, error) {
+		if err := staticBox.findBox("static"); err != nil {
```

Comment:
Handled this way, API endpoints can still be served even if `static` assets can't be found.